grafana/dashboards/stratagem-dashboard-1.json

```diff
@@ -42,11 +42,14 @@
         "alignAsTable": false,
         "avg": false,
         "current": false,
+        "hideEmpty": false,
```

Comment:
What do these params do? Any reason they are not on the other chart?

Reply:
It's probably from setting them to look at the difference and then unchecking them. I'll just take them out. they aren't needed.